docs/UserGuide.md

```diff
@@ -418,6 +450,21 @@ Skills: [Video Editing][Social Media Marketing]
 
 [Back to top <img src="images/back-to-top-icon.png" width="25px" />](#table-of-contents)
 
+---
+### Clearing the applicants : `clear`
+
+Clears the list of applicants
+
+<br>
+
+Format: `clear`
+
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
```

Comment:
Nice update, if you don't mind, could you also add a similar warning for `delete` as it is also another irreversible action?